src/views/User/AvatarCard.tsx

```diff
@@ -92,27 +92,29 @@ export function AvatarCard({
 
     const toggleEdit = () => {
         if (editing) {
-            let promise: Promise<void>;
+            let promise: Promise<any>;
```

Comment:
Nit: `Promise<{ value: boolean }>`

Reply:
I tried really hard to type this, getting as far as 

`let promise: Promise<SweetAlertResult<boolean>> | Promise<{ value: boolean }>;`

however, the thing that Sweetalert returns is just not amenable to clean faking.

Or - if it is, I'd be fascinated to see how.

Reply:
Oh huh, I didn't think optionality would be an issue since `strictNullChecks` is turned off.  Looks like this works instead:

```
Promise<{ value?: boolean }>
               ^
```

`SweetAlertResult<boolean>` looks like the following, so the `value?` needs to match:
```
{
    isConfirmed: boolean
    isDenied: boolean
    isDismissed: boolean
    value?: boolean
    dismiss?: Swal.DismissReason
}
```

Reply:
All hail @benjito, the typing guru 🙇